CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ We use [semantic versioning](http://semver.org/):
 - PATCH version when you make backwards compatible bug fixes.
 
 # Next version
+- [deprecation] _agent_: removed currently unused `test-env` option. Use one of the other testwise coverage modes instead, please.
```

Comment:
We are not deprecating but removing the toggle so IMHO this is a breaking change (iff someone would use it).

```suggestion
- [breaking change] _agent_: removed currently unused `test-env` option. Use one of the other testwise coverage modes instead, please.
```

Reply:
makes sense!